docs/development/CODE_REVIEW_GUIDELINES.md

```diff
@@ -57,6 +57,17 @@ Reviewers (and prospective reviewers) are encouraged to engage in reviews of cod
 
 ### Reviewing
 
+tl;dr Use SOS, TECHDEBT(XXX), TECHDEBT, NIT if you think it'll help the author get context on next steps.
+
+When you leave reviews, if it is unclear from the reviewer to the author, please make sure to mention if:
```

Comment:
I'm struggling a bit with this sentence. Am I interpreting it correctly to be roughly equivalent to something like the following?:

```suggestion
When leaving review comments, consider if any of the following characterizations applies and prefix the comment, respectively:
```